### ISSUE TITLE
Estimations during project creation

### ISSUE THREAD

**Issue by maxceem** [appirio-tech/connect-app#2409]
During initial project creation, there were estimations shown for Visual Design project type. During moving to projectTemplates that block was temporarily hidden, as there was no place to get those estimations from the server and we were going away from hardcoding them in JSON file.

Though after we moved to projectTemplates we still didn't reimplement estimations.

See https://github.com/appirio-tech/connect-app/blob/dev/src/projects/create/components/ProjectOutline.js#L9-L23

**Comment by vikasrohit** [comment]
Thanks @maxceem for capturing this, however, I think we are going do estimate thing later once our catalog stablizes in terms of the phases used for a particular type of project. Once we have stable catalog and prices per phase, we can introduce the estimates again at the time of project creation.

fyi @vic-topcoder @acshields, I think we need to park it for the time being.

**Comment by vic-topcoder** [comment]
Yes - this would be something we'll tackle in the project scope update work. Good catch @maxceem.
